src/soundLoop.js

```diff
@@ -65,6 +65,12 @@ define(function (require) {
 
     this._interval = interval || 1;
```

Comment:
can we do the check on this line? `this._interval = Math.max(interval, 0.01)` ? And then, the question is how can we set the default? We could set the default in the constructor function on line 57, by saying `interval = 1` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters